app/views/views.py

```diff
@@ -1,62 +1,93 @@
+import re
 from flask import Flask, request
 from flask_restplus import Resource, Api, fields
-from ..models.rides import rides
+from ..models.rides import *
+
 
 app = Flask(__name__)
 api = Api(app)
 app.config.from_pyfile('../../config.py')
-obj = rides()
-#models
-new_ride = api.model('new_ride',{
-    'name': fields.String("Name"),
-    'capacity': fields.Integer("capacity"),
-    'time': fields.String("time"),
-    'day': fields.String("day"),
-    'month': fields.String("month"),
-    'year': fields.Integer("year"),
-    'pick up point': fields.String("pick up point"),
-    'Destination': fields.String("destination"),
-    'contact': fields.String("contact")
-    })
-ride_request = api.model('ride_request',{
-    'name': fields.String,
-    'pickup time': fields.String,
-    'destination': fields.String,
-    'pick up point': fields.String,
+obj = new_ride()
```

Comment:
This is not how OOP is used to create instances of `new_ride()`  we use it to create many instances of whatever we are trying to achieve but not a single instance used all over. Or you can take me through your logic of implementation...